charts/pmm/templates/service.yaml

```diff
@@ -21,3 +21,29 @@ spec:
   {{- end }}
   selector:
     {{- include "pmm.selectorLabels" . | nindent 4 }}
+{{- if .Values.ingress.enabled }}
```

Comment:
should it be `{{- if $.Values.ingress.nginxInc }}` ?

Reply:
No, the service is only valid if ingress is enabled. Does not matter the ingress type.